cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -38,6 +38,10 @@
 #include "util/Log.hpp"
 #include "version.h"
 
+namespace {
+std::map<std::thread::id, std::string> g_threadNames;
```

Comment:
Can't acually use thread_local in DistributedSystemImpl anyway since it's used my clicache. C++/CLI used to support [System::ThreadLocalAttribute] like C#. But no longer supported.